src/main/java/uk/gov/hmcts/reform/cwrdapi/service/impl/CaseWorkerServiceFacadeImpl.java

```diff
@@ -143,6 +148,20 @@
         }
     }
 
+    private void validateDuplicateEmailProfiles(List<CaseWorkerDomain> caseWorkerRequest) {
+        List<CaseWorkerDomain> duplicateEmailProfiles = new ArrayList<>();
+        caseWorkerRequest.stream()
+                .map(CaseWorkerProfile.class::cast)
+                .collect(groupingBy(CaseWorkerProfile::getOfficialEmail)).values().stream()
+                .filter(duplicateEmailProfile -> duplicateEmailProfile.size() > 1)
+                .forEach(duplicateEmailProfiles::addAll);
```

Comment:
Really well written piece of code.

Reply:
Thanks. All credit goes to google :D

Reply:
List<CaseWorkerDomain> duplicateEmailProfiles = caseWorkerRequest.stream()
            .map(CaseWorkerProfile.class::cast)
            .collect(groupingBy(CaseWorkerProfile::getOfficialEmail)).values().stream()
            .filter(duplicateEmailProfile -> duplicateEmailProfile.size() > 1)
            .collect(ArrayList::new, List::addAll, List::addAll);
        May be refactor

Reply:
Nice refactor code. Thanks @abhijitdiwan

Reply:
Super one @abhijitdiwan